src/lib.rs

```diff
@@ -175,8 +179,8 @@ impl<F: AsRawFd> File<F> {
     }
 
     /// Converts into a pollable object that supports `std::io::Read`,
-    /// `std::io::Write`, and `tokio_core::io::Io`, making it suitable for
-    /// `tokio_core::io::*`.
+    /// `std::io::Write`, and `tokio_io::io::Io`, making it suitable for
```

Comment:
These should become “`tokio_io::AsyncRead` and `tokio_io::AsyncWrite`”.  `Io` no longer exists.